### PR TITLE
style(ui): sidebar visual update

### PR DESCRIPTION
**Pull request by juliendonck**
Update the sidebar to represent the latest state of the design: 
- Reorder profile / orgs / wallet / etc
- Allow for orgs to overflow

Screenshot 2020-06-15 at 10 52 57

### CODE REVIEW

File: ui/DesignSystem/Component/Sidebar.svelte [hunk 1]
```diff
@@ -20,11 +20,39 @@
...
     flex-direction: column;
     align-items: center;
+    justify-content: space-between;
+  }
+
+  .top {
+    overflow-y: scroll;
+    padding-bottom: 36px;
+    padding-top: 16px;
+  }
```
Comment (rudolfs) [inline ui/DesignSystem/Component/Sidebar.svelte:34]:
```suggestion
  }

```

**Comment by juliendonck** [comment]
@rudolfs I removed Search from the sidebar so that's why the routing test fails (but I don't know how to fix it) but I don't know why the other one does. Could you take a look? Or can I just merge?

**Comment by rudolfs** [comment]
> Could you take a look? Or can I just merge?

I'll have a look when I have some free time. We shouldn't merge broken code.

File: ui/DesignSystem/Component/Sidebar.svelte (hunk 1 above)
Comment (juliendonck) [reply ui/DesignSystem/Component/Sidebar.svelte:35]: Fixed

**Comment by juliendonck** [comment]
@brandonhaslegs good catch, should be fixed now

**Pull request merged**